src/deleteOne.js

```diff
@@ -0,0 +1,16 @@
+import showTasks from './showTasks.js';
+import { setLocalStorage } from './setLocalStorage.js';
+import getTasksFromLocalStorage from './getTasks.js';
+// eslint-disable-next-line import/no-cycle
```

Comment:
**You are missing to enable your ESLint into your pull Request.**
- Enable your Linter after that fix issues if it's needed.